anti-evil-maid.spec.in

```diff
@@ -64,25 +64,22 @@ cp -r systemd $RPM_BUILD_ROOT/usr/lib
 /usr/lib/systemd/system/initrd.target.wants/anti-evil-maid-unseal.service
 /usr/lib/systemd/system/initrd.target.requires/anti-evil-maid-check-mount-devs.service
 
-%define tboot_grub /etc/grub.d/20_linux_tboot /etc/grub.d/20_linux_xen_tboot
+%define trenchboot_grub /etc/grub.d/19_linux_xen_trenchboot
 
 %define refresh \
 dracut --regenerate-all --force \
 grub2-mkconfig -o /boot/grub2/grub.cfg \
 systemctl daemon-reload
 
 %post
-chmod -x %tboot_grub
+chmod -x %trenchboot_grub
 %refresh
 
 %postun
 if [ "$1" = 0 ]; then
     %refresh
-    chmod -f +x %tboot_grub || true
+    chmod -f +x %trenchboot_grub || true
```

Comment:
Something is off with these permissions. The file ends up with 644, instead of 755

Reply:
Removed these lines, no idea what they were supposed to do.

Reply:
refresh regenerates the grub.cfg, so the chmod +x should be before refresh? Or am I wrong? It looks like it was supposed to change the file permissions to avoid creating AEM entry, but give execution permissions to create it later

Reply:
Anyways, the permissions are now correct and the entries are generated properly.